src/pages/ChallengeDetails/ChallengeDetails.jsx

```diff
@@ -0,0 +1,158 @@
+import "./ChallengeDetails.scss";
+import { Link, useParams } from "react-router-dom";
+import DashboardNavbar from "../../components/DashboardNavbar/DashboardNavbar";
+import clock from "../../assets/icons/clock.svg";
+import pen from "../../assets/icons/pen.svg";
+import bulb from "../../assets/icons/lightbulb.svg";
+import challenges from "./challenges.json";
+
+export function ChallengeDetails() {
+
+
+    const params = useParams()
```

Comment:
You can destructure the challengeId out.

```suggestion
    const { challengeId } = useParams()
```

Reply:
ChallengeId is now desctructured